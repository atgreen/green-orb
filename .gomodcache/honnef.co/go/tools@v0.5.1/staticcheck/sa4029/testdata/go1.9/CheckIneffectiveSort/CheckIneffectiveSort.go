package pkg

import "sort"

func fn() {
	type Strings = []string
	var d Strings

	d = sort.StringSlice(d) //@ diag(re`sort\.StringSlice is a type.+consider using sort\.Strings instead`)
}
