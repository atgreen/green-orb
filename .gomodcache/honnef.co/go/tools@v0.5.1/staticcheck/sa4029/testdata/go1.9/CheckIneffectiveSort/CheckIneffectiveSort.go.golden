package pkg

import "sort"

func fn() {
	type Strings = []string
	var d Strings

	sort.Strings(d) //@ diag(re`sort\.StringSlice is a type.+consider using sort\.Strings instead`)
}
